src/components/application_manager/src/request_controller.cc

```diff
@@ -496,12 +495,8 @@ void RequestController::UpdateTimer() {
                    << date_time::DateTime::getmSecs(current_time - end_time)
                    << " Request timeout (sec): "
                    << front->timeout_msec()/date_time::DateTime::MILLISECONDS_IN_SECOND);
-      timer_.Start(0u, false);
+      timer_.Start(0u, true);
     }
-  } else {
-    LOG4CXX_DEBUG(logger_, "Sleep for default sleep time "
-                  << default_sleep_time_ << " milliseconds.");
-    timer_.Start(default_sleep_time_, false);
```

Comment:
@anosach-luxoft, please describe in the commit message why the starts are redundant.